build/templates/_attributes.py.mako

```diff
@@ -104,6 +104,29 @@ class AttributeEnum(object):
         return self._underlying_attribute.__set__(session, value.value)
 
 
+class AttributeBooleanEnum(object):
+    def __init__(
+        self,
+        underlying_attribute_meta_class,
```

Comment:
Why "underlying meta class" and not just "underlying enum"?

Reply:
This class has been removed in my latest commit